site/content/docs/commands/env-deploy.en.md

```diff
@@ -11,7 +11,27 @@ $ copilot env deploy
 
 ```
   -a, --app string    Name of the application.
+      --diff          Compares the generated CloudFormation template to the deployed stack.
       --force         Optional. Force update the environment stack template.
   -h, --help          help for deploy
   -n, --name string   Name of the environment.
+      --no-rollback   Optional. Disable automatic stack
+                      rollback in case of deployment failure.
+                      We do not recommend using this flag for a
+                      production environment.
+```
+
+## Examples
```

Comment:
Can we add a line here and in other deploy commands to tell customers that they can also run `env package --diff` or `svc package --diff` to just see the output without continuing with the deployment or having to press no? I think it's good to mention it since there's not a `--no` or `--force` flag to skip the prompt.